### PR TITLE
feat(ui): adding the ability to view just the output section

### LINKED ISSUE #14924: Add the ability to view the influxdb_v2 output for telegraf
many people already have telegraf configured for writing data in and they just need to add a new output for influxdb_v2. We need to add a way for the user to view just the output config for telegraf instead of having to create a new config and just pulling the output section.

Maybe just a new button next to the create configuration button that just shows the output section and gives the user the option to copy or download. the name of the downloaded file should be `outputs.influxdb_v2.conf`

image

image

### PR DESCRIPTION
**Pull request by drdelambre**
Closes #14924

Added a popup on a route, activated by a button, to help users view just the parts of the telegraf config required for migrating to V2.

Kapture 2019-10-30 at 17 31 50

### CODE REVIEW

File: ui/cypress/e2e/checks.test.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import {Organization} from '../../src/types'
 
+// a generous commitment to delivering this page in a loaded state
+const PAGE_LOAD_SLA = 10000
```
Comment (drdelambre) [inline ui/cypress/e2e/checks.test.ts]: this just unbreaks the test locally for me

File: ui/cypress/e2e/collectors.test.ts [hunk 2]
```diff
@@ -14,48 +17,38 @@ describe('Collectors', () => {
...
   describe('from the org view', () => {
     it('can create a telegraf config', () => {
       const newConfig = 'New Config'
       const configDescription = 'This is a new config testing'
 
-      cy.getByTestID('table-row')
-        .should('have.length', 0)
-        .then(() => {
-          cy.contains('Create Configuration')
+      cy.getByTestID('table-row').should('have.length', 0)
```
Comment (drdelambre) [inline ui/cypress/e2e/collectors.test.ts]: i just moved this test outside of the `then`

File: ui/cypress/e2e/checks.test.ts [hunk 3]
```diff
@@ -18,6 +21,7 @@ describe('Checks', () => {
         cy.visit(`${orgs}/${id}${alerting}`)
       })
     })
+    cy.get('[data-testid="resource-list--body"]', {timeout: PAGE_LOAD_SLA})
```
Comment (drdelambre) [inline ui/cypress/e2e/checks.test.ts]: wait for a test agnostic load event to signal that the tests are ready to run

File: ui/cypress/e2e/collectors.test.ts [hunk 4]
```diff
@@ -76,6 +117,7 @@ describe('Collectors', () => {
         })
 
         cy.reload()
+        cy.get('[data-testid="resource-list--body"]', {timeout: PAGE_LOAD_SLA})
```
Comment (drdelambre) [inline ui/cypress/e2e/collectors.test.ts]: always gotta wait for that page to be ready 😬

File: ui/cypress/e2e/collectors.test.ts [hunk 5]
```diff
@@ -144,111 +173,100 @@ describe('Collectors', () => {
...
 
         // sort by buckets test here
         cy.reload() // clear out filtering state from the previous test
+        cy.get('[data-testid="resource-list--body"]', {timeout: PAGE_LOAD_SLA})
+
         cy.getByTestID('bucket-sorter')
           .click()
           .then(() => {
-            bucketz.sort()
+            // NOTE: this then is just here to let me scope this variable (alex)
```
Comment (drdelambre) [inline ui/cypress/e2e/collectors.test.ts]: or else it's mutation falls outside of the magic promise chain

File: ui/cypress/e2e/checks.test.ts (hunk 1 above)
Comment (zoesteinkamp) [reply ui/cypress/e2e/checks.test.ts]: Makes sense, gives it more time to load

**Pull request merged**